spec/factories/products.rb

```diff
@@ -4,6 +4,7 @@
   factory :product do
     sequence(:name) { |n| "Product#{n}" }
     price { '999.99' }
+    description { lorem }
```

Comment:
what do you think about using factory here? maybe price also should be random

Reply:
Good idea 👍